civictechprojects/static/css/partials/_AboutProject.scss

```diff
@@ -23,309 +27,377 @@
+  margin-left: 10px;
+}
 
-  .AboutProjects-details, .AboutProjects-links, .AboutProjects-files, .AboutProjects-communities, .AboutProjects-team {
-    margin: 10px 5px 10px 15px;
-  }
+.AboutProjects-links, .AboutProjects-files, .AboutProjects-communities, .AboutProjects-team {
+  padding: 10px 5px 10px 15px;
+}
 
-  .AboutProjects-links-icon-container {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content: space-between;
-    width: 90%;
-    margin: auto;
-  }
+.AboutProjects-links, .AboutProjects-files, .AboutProjects-communities {
+  border-bottom: $color-grey-disabled solid 1px;
+}
 
-  .AboutProjects-intro {
-      min-height: 262px;
-  }
+.AboutProjects-links-icon-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 90%;
+  margin: auto;
+}
 
-  .AboutProjects-owner {
-      margin-right: 10px;
-      padding-bottom: 20px;
-  }
+.AboutProjects-intro {
+    min-height: 262px;
+    border-bottom: $color-grey-disabled solid 1px;
+}
 
-  .AboutProjects-owner button {
-    margin-right: 5px;
-  }
+.AboutProjects-introTop {
+  display: flex;
+  min-height: 225px;
+  justify-content: flex-end;
+}
 
-  .AboutProjects-description {
-    padding: 10px 5px;
+.AboutProjects-owner {
+    min-height: 70px;
+}
 
-    h1 {
-      font-size: 1.75rem;
-    }
+.AboutProjects-owner button {
+  margin-right: 13px;
+}
 
-    h3 {
-        margin-bottom: 0;
-        font-size: 20px;
-        font-weight: bold;
-    }
+.AboutProjects-mainColumn {
+  order: 2;
+  width: 70%;
+}
 
-    .AboutProjects-description-issue {
-        color: #888;
-    }
-    .Position-description-link {
-      margin-top: 4px;
-      margin-bottom: 4px;
-    }
+.AboutProjects-description {
+  padding: 10px 5px;
+  width: 70%;
 
+  h1 {
+    font-size: 1.75rem;
   }
 
-  .AboutProjects_tabs {
-    justify-self: flex-start;
-    align-self: flex-end;
-    width: 100%;
-    padding: 0px 0px 0px 15px;
-        a {
-          line-height: 1;
-          margin-bottom: 16px;
-          padding-bottom: 2px;
-          font-size: 24px;
-          margin-right: 40px;
-          color: #888;
-          transition: 300ms;
-          cursor: pointer;
-          &:hover {
-            color: black;
-            text-decoration: none;
-          }
-        }
-        .AboutProjects_aHighlighted {
-          border-bottom: 3px solid $color-orange;
-          padding-bottom: 2px;
-          font-size: 24px;
-          margin-right: 40px;
-          color: #000;
-          transition: 300ms;
-          cursor: pointer;
-          &:hover {
-            color: black;
-            text-decoration: none;
-          }
-        }
+  h3 {
+      margin-bottom: 0;
+      font-size: 20px;
+      font-weight: bold;
   }
 
-  .AboutProjects-description-details {
-    padding: 25px 10px; /*consider max-width instead of padding here */
-
-      #project-details {
-        white-space: pre-wrap;
```

Comment:
We need to keep this styling, otherwise the project description gets rendered as a single line.